client/my-sites/checkout/checkout/composite-checkout-thank-you.ts

```diff
@@ -208,81 +194,8 @@ function getFallbackDestination( {
 	return '/';
 }
 
-function getEligibleDomainFromCart( cart ) {
-	const domainRegistrations = getDomainRegistrations( cart );
-	const domainsInSignupContext = domainRegistrations.filter(
-		domain => domain.extra?.context === 'signup'
-	);
-	const domainsForGSuite = domainsInSignupContext.filter( domain =>
-		canDomainAddGSuite( domain.meta )
-	);
-	return domainsForGSuite.length > 0 ? domainsForGSuite[ 0 ] : null;
-}
-
-function shouldRedirectToGSuiteNudge( {
-	didPurchaseFail,
-	isNewlyCreatedSite,
-	cart,
-}: {
-	didPurchaseFail?: boolean;
-	isNewlyCreatedSite?: boolean;
-	cart: object;
-} ) {
-	if ( isNewlyCreatedSite && ! didPurchaseFail ) {
-		if (
-			! hasGoogleApps( cart ) &&
-			! hasConciergeSession( cart ) &&
-			hasDomainRegistration( cart )
-		) {
-			return true;
-		}
-	}
-	return false;
-}
-
-function getRedirectUrlForPlanBumpOffer( { pendingOrReceiptId, cart, siteSlug } ) {
-	if ( hasPersonalPlan( cart ) ) {
-		if ( 'variantShowPlanBumpOffer' === abtest( 'showPlanBumpVsGsuite' ) ) {
-			return `/checkout/${ siteSlug }/offer-plan-upgrade/premium/${ pendingOrReceiptId }`;
-		}
-	}
-	return null;
-}
```

Comment:
@victorespigares: I think this method and some of the code that relates to https://github.com/Automattic/wp-calypso/pull/38680 should have been removed in https://github.com/Automattic/wp-calypso/pull/39296. Note I'm not sure how changes to the existing and new checkouts are supposed to be handled/synced, maybe @payton or @nbloomf could clarify that.